packages/vaadin-lumo-styles/components/avatar.css

```diff
@@ -13,7 +13,8 @@
     lumo_components_avatar;
 }
 
-:is(:root, :host) {
```

Comment:
Would changing it to `:where` have the same effect you're looking for?

Reply:
Hmm, that could work! Let me try